atlas/atlas.star

```diff
@@ -0,0 +1,175 @@
+"""
+A URL is an absolute location within the atlas.
+"""
+scalar URL
+
+"""
+Names are dotted paths of GraphQL identifiers, like `spotify.User`
```

Comment:
Is there a length constraint on this? ie, "we only allow 4 levels of nesting/paths"?

Additionally, when you say "identifier" is that in reference to [`ID`](https://spec.graphql.org/draft/#sec-ID) or maybe [`Name`](https://spec.graphql.org/draft/#Name)? (`Name` would make more sense to me since that would limit the characters possible)

It seems like we'll

Reply:
(also, I think a _maybe_ better name here is `Path` so as to not overshadow the graphql spec but I'm not wed to that opinion in the least)